src/discparse.py

```diff
@@ -217,30 +217,39 @@ async def get_dvdinfo(self, discs):
             path = each.get('path')
             os.chdir(path)
             files = glob(f"VTS_*.VOB")
-            files.sort()
-            filesdict = [[]]
+            # Switch to natural sorting
+            files = natsorted(files)
```

Comment:
do you have an example where natsort would result in a different sort, and thus be worth the additional dependency? As far as I'm aware the file naming structure of "VTS_(00-99)_(0-9).VOB" shouldn't have any issues with a normal sort

Reply:
I'm genuinely trying to remember *why* I did that. I was seeing random VOBs get picked first and (probably badly) assumed that it was an issue with the sorting. I can try rolling back and seeing if it still functions as expected with the other changes.